src/app/not-found/not-found.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+declare var particlesJS: any;
+
+@Component({
+  selector: 'mifosx-not-found',
+  templateUrl: './not-found.component.html',
+  styleUrls: ['./not-found.component.scss']
+})
+export class NotFoundComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+    particlesJS.load('particles-js', '/assets/data/particles.json', function() {
```

Comment:
Add the file into the assets folder directly, no need to create a new folder data to store it.